compass/ocean/tests/isomip_plus/viz/plot.py

```diff
@@ -677,13 +686,15 @@ def plot_layer_interfaces(self, figsize=(9, 5)):
             plt.plot(1e-3 * X[0, :], self.zBotSection, 'g')
 
             ax.autoscale(tight=True)
+            x1, x2, y1, y2 = 420, 470, -650, -520
+            xlim = [min(x1, 1e-3*numpy.amin(X)), 1e-3*numpy.amax(X)]
+            plt.xlim(xlim)
```

Comment:
The inset was weird if the full x axis didn't include x = 420 km (i.e. for non-thin-film cases).  So I'm making sure it includes the full inset box.